build/yamls/windows/base/conf/antrea-agent.conf

```diff
@@ -38,18 +38,32 @@ featureGates:
 # Enable metrics exposure via Prometheus. Initializes Prometheus metrics listener.
 #enablePrometheusMetrics: true
 
-# Provide flow collector address as string with format <IP>:<port>[:<proto>], where proto is tcp or udp.
+# Provide the IPFIX collector address as a string with format <HOST>:[<PORT>][:<PROTO>].
+# HOST can only be IP right now because there is a DNS resolution issue in current Windows support.
 # IP can be either IPv4 or IPv6. However, IPv6 address should be wrapped with [].
-# This also enables the flow exporter that sends IPFIX flow records of conntrack flows on OVS bridge.
-# If no L4 transport proto is given, we consider tcp as default.
+# If PORT is empty, we default to 4739, the standard IPFIX port.
+# If no PROTO is given, we consider "tcp" as default. We support "tcp" and "udp"
+# L4 transport protocols.
 #flowCollectorAddr: ""
 
-# Provide flow poll interval as a duration string. This determines how often the flow exporter dumps connections from the conntrack module.
-# Flow poll interval should be greater than or equal to 1s (one second).
+# Provide flow poll interval as a duration string. This determines how often the
+# flow exporter dumps connections from the conntrack module. Flow poll interval
+# should be greater than or equal to 1s (one second).
 # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 #flowPollInterval: "5s"
 
-# Provide flow export frequency, which is the number of poll cycles elapsed before flow exporter exports flow records to
-# the flow collector.
-# Flow export frequency should be greater than or equal to 1.
-#flowExportFrequency: 12
+# Provide the active flow export timeout, which is the timeout after which a flow
+# record is sent to the collector for active flows. Thus, for flows with a continuous
+# stream of packets, a flow record will be exported to the collector once the elapsed
+# time since the last export event is equal to the value of this timeout.
+# Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+#activeFlowExportTimeout: "60s"
```

Comment:
We are moving the default to 30s as part of this PR: #1949 
Could you update that here to reflect the same?

Reply:
Sure, updated.